tests/proposals.rs

```diff
@@ -27,7 +27,7 @@ async fn test_proposal() -> anyhow::Result<()> {
                 "proposal_body_version": "V0",
                 "name": "another post",
                 "description": "some description",
-                "category": "Marketing",
+                "category": "Bounty",
```

Comment:
I would not change these. Instead, I would explicitly initialize the categories, so we don't need to rely on default categories list